src/Api/CachedIdentifiersExtractor.php

```diff
@@ -46,60 +47,66 @@ public function __construct(CacheItemPoolInterface $cacheItemPool, IdentifiersEx
      */
     public function getIdentifiersFromItem($item): array
     {
-        $identifiers = [];
-        $resourceClass = $this->getObjectClass($item);
-
-        $cacheKey = self::CACHE_KEY_PREFIX.md5($resourceClass);
-
-        // This is to avoid setting the cache twice in the case where the related item cache doesn't exist
-        $cacheIsHit = false;
+        $keys = $this->getKeys($item, function ($item) use (&$identifiers) {
+            return $identifiers = $this->decorated->getIdentifiersFromItem($item);
+        });
```

Comment:
Using a reference to an unknown variable and declaring it in a closure looks cool to me haha :sweat_smile: